plugins/primus_test/lisp/memcheck.lisp

```diff
@@ -24,6 +24,12 @@
 (require incident)
 
 ;; Public Interface
+
+(defun memcheck-is-allocated (heap ptr)
```

Comment:
I would name it `memcheck-is/was-allocated` or even better, `memcheck-is-tracked` to prevent future confusion.